.changeset/swift-panthers-play.md

```diff
@@ -0,0 +1,5 @@
+---
+'@accounts/mongo-sessions': minor
```

Comment:
I think that this change should also update the mongo package as AccountsMongoOptions was changed

Reply:
Resolved in 710f357